apps/calendar/elements/multi_day.html

```diff
@@ -1,7 +1,7 @@
 <element name="multi-day" extends="section">
   <template>
     <div class="md__sticky">
-      <span class="md__all-day gaia-icon icon-calendar-allday" aria-hidden="true" data-l10n-id="all-day"></span>
+      <span class="md__all-day" data-icon="all-day" aria-hidden="true" data-l10n-id="all-day"></span>
```

Comment:
this change broke the selector used on `js/views/multi_day.js`, you also need to update it:

```diff
diff --git a/apps/calendar/js/views/multi_day.js b/apps/calendar/js/views/multi_day.js
index 2f93490..1ec72c5 100644
--- a/apps/calendar/js/views/multi_day.js
+++ b/apps/calendar/js/views/multi_day.js
@@ -69,7 +69,7 @@ MultiDay.prototype = {
   },
 
   get allDayIcon() {
-    return this.element.querySelector('.icon-calendar-allday');
+    return this.element.querySelector('.md__all-day');
   },
 
   onactive: function() {
```